Microsoft.Toolkit.Uwp.UI.Controls/Microsoft.Toolkit.Uwp.UI.Controls.csproj

```diff
@@ -0,0 +1,43 @@
+    <ProjectReference Include="..\Microsoft.Toolkit.Uwp.UI.Controls.Layout\Microsoft.Toolkit.Uwp.UI.Controls.Layout.csproj" />
+  </ItemGroup>
+  
+  <!--
+    Required workaround for ProjectReference inclusion of the Controls package
+    The UWP project system is including the Controls resources in the pri file because 
+    it doesn't know it'll be an independent package later during packing.
+    Therefore, we need to remove these extra resources in the PRI pipeline so the 
+    Markdown pri file is properly generated and doesn't include duplicate references to Control resources.
+  -->
+  <Target Name="RemoveUnwantedPri" AfterTargets="GetPackagingOutputs">
+    <!--<Message Text="Files Before: @(PackagingOutputs)" Importance="high" />-->
+    <ItemGroup>
+      <PackagingOutputs Remove="@(PackagingOutputs)" Condition="'%(PackagingOutputs.Filename)%(PackagingOutputs.Extension)' == 'Microsoft.Toolkit.Uwp.UI.Controls.Core.pri'" />
+      <PackagingOutputs Remove="@(PackagingOutputs)" Condition="'%(PackagingOutputs.Filename)%(PackagingOutputs.Extension)' == 'Microsoft.Toolkit.Uwp.UI.Controls.Primitives.pri'" />
+      <PackagingOutputs Remove="@(PackagingOutputs)" Condition="'%(PackagingOutputs.Filename)%(PackagingOutputs.Extension)' == 'Microsoft.Toolkit.Uwp.UI.Controls.Media.pri'" />
+      <PackagingOutputs Remove="@(PackagingOutputs)" Condition="'%(PackagingOutputs.Filename)%(PackagingOutputs.Extension)' == 'Microsoft.Toolkit.Uwp.UI.Controls.DataGrid.pri'" />
+      <PackagingOutputs Remove="@(PackagingOutputs)" Condition="'%(PackagingOutputs.Filename)%(PackagingOutputs.Extension)' == 'Microsoft.Toolkit.Uwp.UI.Controls.Markdown.pri'" />
+      <PackagingOutputs Remove="@(PackagingOutputs)" Condition="'%(PackagingOutputs.Filename)%(PackagingOutputs.Extension)' == 'Microsoft.Toolkit.Uwp.UI.Controls.Layout.pri'" />
+      <PackagingOutputs Remove="@(PackagingOutputs)" Condition="$([System.String]::new('%(PackagingOutputs.TargetPath)').StartsWith('Microsoft.Toolkit.Uwp.UI.Controls.Primitives\'))" />
+      <PackagingOutputs Remove="@(PackagingOutputs)" Condition="$([System.String]::new('%(PackagingOutputs.TargetPath)').StartsWith('Microsoft.Toolkit.Uwp.UI.Controls.Core\'))" />
+      <PackagingOutputs Remove="@(PackagingOutputs)" Condition="$([System.String]::new('%(PackagingOutputs.TargetPath)').StartsWith('Microsoft.Toolkit.Uwp.UI.Controls.Media\'))" />
+      <PackagingOutputs Remove="@(PackagingOutputs)" Condition="$([System.String]::new('%(PackagingOutputs.TargetPath)').StartsWith('Microsoft.Toolkit.Uwp.UI.Controls.DataGrid\'))" />
+      <PackagingOutputs Remove="@(PackagingOutputs)" Condition="$([System.String]::new('%(PackagingOutputs.TargetPath)').StartsWith('Microsoft.Toolkit.Uwp.UI.Controls.Markdown\'))" />
+      <PackagingOutputs Remove="@(PackagingOutputs)" Condition="$([System.String]::new('%(PackagingOutputs.TargetPath)').StartsWith('Microsoft.Toolkit.Uwp.UI.Controls.Layout\'))" />
```

Comment:
@michael-hawker I don't have a lot of knowledge on how the original "RemoveUnwantedPri" code in other projects but, this code seemed to have fixed issues when including the package in a test project for me, let me know if this code here makes sense.

Also there has to be a better way to write this, I am not that familiar with what syntax is available here. I anyone wants to help me clean this up.

Reply:
I wonder if this is because we're using ProjectReference,.. @azchohfi could we just use PackageReference here and see if that removes the problem? Or that's going to be an issue, eh? As then we'd have to build it after the others?

But otherwise, yes I think we need these to work-around how the build system works. Ideally these packages don't contain any content, so we may need to look at the final nuget package built in the end.

Reply:
It would make our life a mess to sync the version. I don't see why we would need these, as the package itself should not contain anything. If adding the `<IncludeBuildOutput>false</IncludeBuildOutput>` don't fix it, which I think it will, we could always just use a nuspec.